src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -48,10 +49,7 @@ def process_job_results(
 
             if len(unwritten_results) > batch_size:
                 written_results, unwritten_results = results.write_results_batch(
-                    output_directory,
-                    written_results,
-                    unwritten_results,
-                    batch_size,
+                    output_directory, written_results, unwritten_results, batch_size
```

Comment:
Did you delete the trailing comma here?  That's what induces the whitespace noise.

Reply:
I've been having issues with black/isort so I'll add this back in.